python/Scripts/OIIOLoader.py

```diff
@@ -0,0 +1,429 @@
+"""
```

Comment:
Just to match the surrounding naming conventions, I'd recommend naming this script `oiio_loader.py`.